### ISSUE TITLE
md-autocomplete: number of matching items and the selected one are not announced by the screen reader at the same time.

### ISSUE THREAD

**Issue by wnwnzhou** [angular/material#9603]
**Actual Behavior**:
 -  `What is the issue? *`
In the case when the autocomplete dropdown shows a list of matching item, and a auto selected one in the list:
Sometimes the screen reader only announces the '# of matches available', or only the auto selected item.

 -  `What is the expected behavior?`
Should announce both the # of matches item and the auto selected one together.
I took a look into the code, and found that the message announce by the screen reader is put in <aria-status> tag which has aria-live. But by default aria-live only catches the changed content. I suggest to add aria-atomic="true" attribute to <aria-status>, so that all the messages in it can be announced together when the change is detected.

**Angular Versions**: *
 -  `Angular Version:` 1.5.9
 -  `Angular Material Version:` 1.1.1

**Additional Information**:
 -  `Browser Type: *` Chrome
 -  `Browser Version: *`
 -  `OS: *` 
 -  `Stack Traces:`

----
Shortcut to create a [new CodePen Demo](http://codepen.io/team/AngularMaterial/pen/bEGJdd).
Note: `*` indicates required information. Without this information, your issue may be auto-closed.

> Do not modify the titles or questions. Simply add your responses to the ends of the questions.
  Add more lines if needed.
